scenes/player/Player2d.gd

```diff
@@ -30,5 +30,13 @@ func get_input():
 
 func _physics_process(_delta):
 	get_input()
-	
 	move_and_slide()
+
+
+
+
+func _on_area_2d_area_entered(area):
+	print(health)
```

Comment:
We should probably add a check to make sure the area we are entering is actually an enemy, just in case we add other areas to the game.